Sources/AsyncAlgorithms/Rethrow.swift

```diff
@@ -26,7 +26,7 @@ extension _ErrorMechanism {
     fatalError("materialized error without being in a throwing context")
   }
   
-  internal func _rethrowGet() rethrows -> Output {
+  @usableFromInline internal func _rethrowGet() rethrows -> Output {
```

Comment:
we should avoid this; the `_ErrorMechanism` is a work-around for a compiler feature that is not yet there so ideally it should not be exposed into ABI.

Reply:
removed :-)